clusterloader2/testing/load/config.yaml

```diff
@@ -99,11 +99,13 @@ tuningSets:
 - name: Sequence
   parallelismLimitedLoad:
     parallelismLimit: 1
+{{if not $IS_SMALL_CLUSTER}}
```

Comment:
Hmm - actually - why this is failing validation?
Looking at stuff above:
```
totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) (MultiplyInt 2 .Nodes)}
```
So it's at minimum 2*500=1000
```
$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment
```

and we defide by numberOfNodes.

So it should be pretty big. So why it is failing?

Reply:
Failure takes place because of [this](https://github.com/kubernetes/perf-tests/blob/309e9ff97cfa9b3707ff6c01b5a8b7056bebcf22/clusterloader2/testing/load/config.yaml#L85) line which happens later in the config:
```
{{$schedulerThroughputNamespaces := IfThenElse $IS_SMALL_CLUSTER 0 $schedulerThroughputNamespaces}}
```

Reply:
Thanks.

So actually - can we remove this line instead?
I don't see anything super bad in creating those namespaces (they will not be used later).
Alternatively, we can set it to 1 instead of 0 in this if.

Reply:
Yup it's not such a big deal. It would bloat the cluster with namespaces and if the suite fails midway, all the namespaces remain and you'll have to clean them up. I think setting to 1 will be okay.